app/components/search-facet-taxonomy.js

```diff
@@ -99,6 +100,20 @@ export default Ember.Component.extend(Analytics, {
             this._expand(item);
         }
     },
+    _expandDefault(){
+        let topLevelItem = this.get('topLevelItem');
+        if (topLevelItem.length <= 3){
+            topLevelItem.forEach(item => {
```

Comment:
I think I like the `=> {` better than `=>{` below, nit picky things, but might as well fix while in there.